### ISSUE TITLE
No MOT configurations specified

### ISSUE THREAD

**Issue by fcornelius** [ad-freiburg/pfaedle#6]
I get the following error when trying the example from the `README`
```
pfaedle -D -x freiburg-regbez-latest.osm .
[2019-06-27 00:25:56.865] ERROR: No MOT configurations specified and no implicit configurations found, see --help.
```
I get the same for all other osm files I have tried it with. Manually setting a mot type does not change the outcome.
I'm running `cmake` with gcc 4.9.4 and g++ 9.1.0 on mac os 10.14.
Any idea what could cause this error?

**Comment by patrickbr** [comment]
pfaedle needs a configuration file (a default one, pfaedle.cfg is provided in the repository). By default, pfaedle checks the following directories for configuration files, in that order.

1. `CMAKE_INSTALL_PREFIX/etc/pfaedle/pfaedle.cfg`
2. `$HOME/.config/pfaedle/pfaedle.cfg`
3. `CWD/pfaedle.cfg`

Alternatively, one (or multiple) config files can be given via the -c flag.

So I guess the simplest solution in your case would be to either move the `pfaedle.cfg` in this repo to `$HOME/.config/pfaedle/pfaedle.cfg` or, alternatively, to just call pfaedle like this:

    pfaedle -c path/to/pfaedle.cfg -D -x freiburg-regbez-latest.osm .

I am leaving this issue open as this is not the intended behavior. After `make install`, the file `pfaedle.cfg` should lie somewhere where pfaedle can find it. However, I haven't gotten around to testing this on macOS.

**Comment by fcornelius** [comment]
Thanks for the super quick response!
You're right, the config somehow ended up in `/usr/local/etc/pfaedle/pfaedle.cfg` with pfaedle being installed to `/usr/local/bin/pfaedle`. 
Moving it into one of the scanned directories fixed it for me 👍

**Comment by patrickbr** [comment]
Just reproduced this bug on a Mac with the help of @manonthegithub and @naetherm. Should be fixed with bfca604ab56272348ea7fdb3bddb6c03e9f089e6 :)
